Reusable_Components_Doc.md

```diff
@@ -22,3 +22,124 @@ _____
+* **Input/Props**: {title, isEditing, setEditingStatus}
+* **Currently used in**: emergencyContacts, Manager
+* **Will be used in**: Tenants (figma pg 30), Properties (figma pg 49)
+* **Additional info**: While isEditing and setEditingStatus can be initiated in the parent scope with useState(), for convience they are in a hook that can be imported by name from the same file. (i.e <import TitleAndPen, { useEditingStatus } from '../components/TitleAndPen';>) \
+You would then call it like this: <const { isEditing, setEditingStatus } = useEditingStatus()>
+_____
+
+### **Name**: ToggleEditTable
+* **File path**: "src/components/ToggleEditTable"
+* **Input/Props**: {isEditing, tableData, submitHandler, cancelHandler, validationSchema}
+* **Currently used in**: Manager
+* **Will be used in**: emergencyContacts (Figma pg. 22), Tenants (Figma pg. 30), Properties (figma pg. 49)
+* **Additional info**: Below is a breakdown of each prop to hopefully help make this easy to use! :)
+
+#### isEditing
+A boolean that controls if an uneditable table should display, or if the Formik form should display, allowing users to edit the tableData.
+
+#### tableData
+tableData is an array of objects. Below is the shape of tableData, and it's array of objects, with a description of each. Getting this correct is super important to making the component work correctly.
+
+```
+tableData: PropTypes.arrayOf(
+    PropTypes.shape({
+      key: PropTypes.string.isRequired,
+      label: PropTypes.string.isRequired,
+      value: PropTypes.isRequired,
+      inputType: PropTypes.string.isRequired,
+      placeholder: PropTypes.string,
+    })
+  )
+```
+
+* **key:** A key name for the field, which should be camel cased. ***Important!*** Make sure that the key of each field you supply matches the key you provide to validationSchema (see below).
+* **label:** This string will be used for the input's label
+* **value:** This is the initial value for the input. If you wish for there to not be an initial value, use an empty string.
+* **inputType:** This identifies what the input type will be (checkbox, number, text, etc.)
+* **placeholder:** This is the placeholder text that will be used by the input.
+
+#### submitHandler / cancelHandler
+A "Save" and "Cancel" button are generated as part of this component. The submitHandler is sent to the "Save" button's onClick, and cancelHandler is sent to the "Cancel" button's onClick.
+
+submitHandler likely will need to make an API call to save data on the backend, and cancelHandler generally just needs to set isEditing to false.
+
+#### validationSchema
+Use [Yup](https://formik.org/docs/guides/validation) to create a validationSchema (this is recommended by Formik). ***Important!*** Make sure that the key of each item matches the key of each field that you supply into tableData.
+
+Note, once Anna's reusable Form component (PR #309) is merged into development, this component should be refactored to utilize it.
```

Comment:
Anna's component was just merged! Did you want to update this documentation now and add the InfoFormField into this doc? Otherwise we can create an issue for it and remove this line

Reply:
Talked with Darren, we'll come back to this in a different issue